_rules/printable-characters-shortcut-ffbc54.md

```diff
@@ -0,0 +1,362 @@
+---
+id: ffbc54
+name: No keyboard shortcut uses only printable characters
+rule_type: atomic
+description: |
+  This rule checks that if keyboard shortcuts are implemented using only printable characters, then there is a mechanism to disable the shortcut, or to remap the shortcut to use one or more non-printable character keys, or the shortcut for a user interface component is only available when that component has focus.
+accessibility_requirements:
+  wcag21:2.1.4: # Character Key Shortcuts (A)
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+acknowledgments:
+  authors:
+    - João Vicente
+    - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [keyboard event][] for which all of the following is true: 
+- the event's attribute `key` is a [printable character][] key; and
+- the event's method `getModifierState` returns `false` for each of the [valid modifier keys][]; and
+- the event causes [changes in the content][changes in content] of the [HTML document][].
+
+## Expectation
+
+For each test target at least one of the following is true:
+- (**remap**:) there is at least one [set of clearly labeled instruments][] to [block events][blocked event] that use the [same key][same key events] as the test target and whose `getModifierState` method returns `false` for each of the [valid modifier keys][]; or
+- (**focus**:) the [event target][] has a [semantic role][] that inherits from the [abstract role](https://www.w3.org/TR/wai-aria/#abstract_roles) of `widget`.
```

Comment:
There is one edge case we seem to not handle…
If there is a remap mechanism to block the event unless the widget has focus.

OTOH, strict reading of the SC seems to not allow that… So I guess this is OK…